### DISCUSSION TITLE
Does anyone else find the lack of a start block confusing?

### DISCUSSION THREAD

**Discussion by freemansoft** [BIPES/BIPES#168] [General]
I have two flows on my blocky canvas.  There is no way to determine which one will run first because there is no "startup" block. 

I played a little with UI flow and the wrap the "main" function in a component.

**Comment by jetannenbaum** [comment]
Hi Joe,

   I generally save my flows so I only have one flow on the canvas at a
time.  Do you have a particular use case that would require two flows on
the same canvas?  Assuming you do, once the first flow starts, when would
the second one start?  Perhaps a timer is the way to go?

   Thanks,

     Jet

**Comment by freemansoft** [comment]
You already support wrappers for _functions_, _timers_, and for the _Project Info block_. I'm suggesting you do it for "main()" to be consistent.  

Having two flows was an accident.  It didn't complain and it ran them in random order.  That could be problematic for a new developer.  You can leave stuff hanging in some corner of the canvas. Everyone will have effectively two flowssince everyone gets a  _Project Info_ segment.

I'm not a big fan of the app UIFlow, for the M5Stack products, but it has the main program wrapper.  That program is also interesting because it lets you drag and drop labels on a screen mockup and then generates those fields as variables that can be updated.  They only support built-in displays for their products so it isn't as open-ended as it sounds.
